snuba/subscriptions/worker.py

```diff
@@ -189,7 +191,9 @@ def callback_func(
                                 "consistent": str(is_consistent_query),
                                 "primary": primary_result,
                                 "other": result.result,
-                                "query": request.query,
+                                "query": request.query.get_all_expressions(),
```

Comment:
This might be hard to read since it just slams all of the expressions into one big array, with no delineation about which expressions are part of what clause. It might be worth breaking up, e.g. `"selected": request.query.get_selected_columns(), "condition":...`.